src/render.js

```diff
@@ -5,9 +5,32 @@ const RenderPosition = {
   AFTEREND: 'afterend',
 };
 
-const renderTemplate = (container, template, place) => {
-  container.insertAdjacentHTML(place, template);
+const render = (container, element, place) => {
+  switch (place) {
+    case RenderPosition.BEFOREBEGIN:
+      container.before(element);
+      break;
```

Comment:
```suggestion
    case RenderPosition.BEFOREBEGIN: {
      container.before(element);
      break;
      }
```
всегда добавляй кавычки кейсам